### ISSUE TITLE
Fix for lost songs

### ISSUE THREAD

**Issue by Dschingan** [Malopieds/InnerTune#116]
### Checklist

- [X] I've checked that there is no other issue about this feature request.
- [X] This issue contains only one feature request.
- [X] The title of this issue accurately describes the feature request.

### Feature description

It would be nice if there was a check button next to sync and when we click it we could compare the number of songs before syncing with YTM. I don't know if it would be difficult, but it would be nice if we could see the names of the songs that are not in YTM playlist but in the InnerTune playlist before synchronization.

### Why do you want this feature?

When songs with videos (yt song videos) are removed from YouTube Music, the song disappears because its name and picture are not visible and we have no way of knowing which song it is. So before synchronizing the playlist in InnerTune, I check that the number of songs is the same in both YTM and InnerTune. This makes it difficult to know which song has been removed.

**Comment by Malopieds** [comment]
What about when you sync, a pop up appears and it tells you if there are more, less or the same number of songs. After that you can decide if you want to sync or abort?

**Comment by Dschingan** [comment]
> Senkronizasyon yaptığınızda, bir açılır pencere belirir ve size şarkıların daha fazla mı, daha az mı yoksa aynı sayıda mı olduğunu söyler. Bundan sonra senkronize etmek mi yoksa iptal etmek mi istediğinize karar verebilirsiniz.

Yes, that's exactly what I meant. It would also be nice if InnerTune would show the name of the missing song in the YTM playlist

**Comment by Malopieds** [comment]
Ok, I'll see what I can do. I don't have much time this period with exam and school.

**Comment by Golem642** [comment]
And also tell which songs have been removed before syncing, so you know what to expect once you synced
